tests/ssl-opt.sh

```diff
@@ -802,10 +782,6 @@ requires_openssl_tls1_3_with_ffdh() {
 # skip next test if openssl cannot handle ephemeral key exchange
 requires_openssl_tls1_3_with_compatible_ephemeral() {
     requires_openssl_next
-
-    if !(is_config_enabled "PSA_WANT_ALG_ECDH"); then
```

Comment:
I'm not sure about this. AFAIK `PSA_WANT_ALG_ECDH` is not going to be removed so this should stay I think